src/main/java/frc/robot/subsystems/conveyor/commands/bits/TestColorSensor.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.conveyor.commands.bits;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+public class TestColorSensor extends SequentialCommandGroup {
+
+    public TestColorSensor(Conveyor conveyor) {
+        addCommands(
+                new RunCommand(() -> System.out.println("Enter blue cargo")).withTimeout(4),
+                new RunCommand(() -> checkColorBlue(conveyor)).withTimeout(10),
+                new RunCommand(() -> System.out.println("Enter red cargo")).withTimeout(4),
+                new RunCommand(() -> checkColorRed(conveyor)).withTimeout(10)
```

Comment:
You can run the `checkColor{color}` without a timeout as it involves human interaction with the robot, and finish it when the robot senses the color. (you can do that by using `RunCommand(...).withInterrupt(<condition>)`